### PR TITLE
Update Chinese Strings

### PR DESCRIPTION
**Pull request by sr093906**

### CODE REVIEW

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -36,6 +36,9 @@
     <string name="pref_default_search">默认搜索引擎</string>
     <string name="pref_default_search_summary">总是在列表的底部显示特定的搜索引擎</string>
     <string name="pref_default_search_none">（无）</string>
+    <string name="pref_string_match_strategy">文本搜索匹配样式</string>
+    <string name="pref_string_match_strategy_substring">正常</string>
```
Comment (nhirokinet) [inline app/src/main/res/values-zh-rCN/strings.xml:40]: I don't know about Chinese, but is it OK to use "正常" to just default setting, not "correct state"? Sorry if my mistake.

"正常" in Japanese means "correct state" and this option is not "correct state". Maybe different in Chinese?

Comment (sr093906) [reply app/src/main/res/values-zh-rCN/strings.xml:40]: @nhirokinet I think the 'normal' style here means 'full-text match' actually. But if you insist using 'normal', then as a native Chinese, I am sure the translation is right.

Comment (nhirokinet) [reply app/src/main/res/values-zh-rCN/strings.xml:40]: Thank you for confirmation. Yes, this is technically full-text match actually. Actually I wasn't sure how to express it as app setting, but come to write "normal".

Now I've got sure that there is no misunderstanding here.

**Comment by nhirokinet** [comment]
Thank you for many times! I'm planning this code to be released as 1.0.11 after some testing.

If you were willing to translate app explanation on F-Droid and Google Play Store, I'd be glad if you could make a pull request creating `fastlane/metadata/android/zh-CN/full_description.txt` and `fastlane/metadata/android/zh-CN/short_description.txt`,
which is read from F-Droid, and I can copy and paste to Google Play Store. In this case, I will add screenshots of Chinese version with the same phone as English and Japanese screenshots after 2 text files are added.

**Comment by sr093906** [comment]
@nhirokinet Will do it on Monday. Now i will go to bed. :)

**Pull request merged**